src/System.Device.Gpio/System/Device/Gpio/Drivers/UnixDriver.cs

```diff
@@ -32,18 +34,39 @@ public static UnixDriver Create()
         }
 
         UnixDriver? driver = null;
+
+        if (TryCreate(() => new LibGpiodDriver(0), out driver))
```

Comment:
do we need to pass in 0 here? shouldn't this be `()` and below

Reply:
We should consider moving this logic to some other place. We shouldn't have detection logic all over the place that works differently each time. 

I would consider obsoleting this, but we can discuss.